.asf.yaml

```diff
@@ -38,7 +38,8 @@ github:
   protected_branches:
     master:
       required_status_checks:
-        strict: true
+        contexts:
+          - Dead Link Checker
```

Comment:
This might be `CheckDeadLinks`.

```suggestion
          - CheckDeadLinks
```

Reply:
So it uses job's name?

Reply:
It's called context name in GitHub, to get this exact name, open a merged PR like https://github.com/apache/dolphinscheduler-website/pull/674, click the "view details" and copy that name

<img width="858" alt="Screen Shot 2022-02-09 at 21 10 12" src="https://user-images.githubusercontent.com/15965696/153207772-c8625a83-d065-4ec8-816b-20864ee6983a.png">

Reply:
Thanks for clarification 👍